<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2078</width>
    <height>1027</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-40</y>
      <width>2104</width>
      <height>1089</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Forca_img/telafundo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>1600</width>
      <height>967</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Forca_img/Rectangle 14.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>1426</width>
      <height>223</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Forca_img/Group 2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>1426</width>
      <height>223</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Forca_img/Group 3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>730</y>
      <width>1426</width>
      <height>223</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Forca_img/Group 4.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="avancar_2">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>860</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Forca_img/ButtomPross.png</normaloff>Forca_img/ButtomPross.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="voltar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>860</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Forca_img/Default.png</normaloff>Forca_img/Default.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>536</width>
      <height>99</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Forca_img/Component 12.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2078</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
